### ISSUE TITLE
Empty links for profiles with no affiliations

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/knowledge-commons-profiles#580]
Area of Site: Profiles
URL: https://profile.hcommons-test.org/members/

**Description**
For member profiles that lack an affiliation, their results on the Members page contain anchor tags (`<a>`) with no discernible text nested in `div` elements with the `.aff` class attribute. It seems these links would ordinarily be labeled with an affiliated group name, but, because there's no affiliation, the links are empty and create accessibility issues. 

<img width="1554" height="474" alt="Screenshot showing an example of a member profile result on the Members page containing a hyperlink with no text" src="https://github.com/user-attachments/assets/f3399285-b45b-45a2-b0e6-8e7c1a5eecdf" />

**Solution**
Conditionally render `<div class="aff">` containers of affiliation links. If there's no affiliation, do not render a div containing this anchor tag at all.
